cli/sawtooth_cli/main.py

```diff
@@ -184,3 +184,7 @@ def main_wrapper():
     except:
         traceback.print_exc(file=sys.stderr)
         sys.exit(1)
+
+
+if __name__ == '__main__':
```

Comment:
This is provided by the bin/sawtooth script in the repo, or by the auto-generated wrapper script when installed. This non-executable file doesn't need it.

Reply:
Done. This was removed